8Q.py

```diff
@@ -0,0 +1,13 @@
+def counter():
+    word=input("Enter a string: ")
+    count_upper=0
+    count_lower=0
+    for i in word:
+        if i.isupper():
+            count_upper+=1
+        else:
+            count_lower+=1
+    print("Lower letter: "+str(count_lower))
+    print("Upper letter: " + str(count_upper))
+
+counter()
```

Comment:
This code also has to be fixed!!! Because here is my `input` `123 biR Sayi MIDIR?` and this outputs `Lower letter: 12`
`Upper letter: 7` respectively. Is it True? Absolutely not, and here is the right output:
Lover Letter : 5
Upper Letter : 7
Other: 7 (not necessary).